src/components/containers/menu/MenuPanel.js

```diff
@@ -27,29 +26,23 @@ const StyledChildrenContainer = styled.div`
 
 const MenuPanel = (props, context) => {
   const { children, isOpen, closeFunction } = props;
-
   return (
-    <ThemeProvider theme={defaultTheme}>
-      <StyledPanel isOpen={isOpen}>
-        <Spacer>
-          <StyledDismissButton onClick={closeFunction} />
-        </Spacer>
-        <StyledChildrenContainer inline={context.inline}>
-          {children}
-        </StyledChildrenContainer>
-      </StyledPanel>
-    </ThemeProvider>
+    <StyledPanel isOpen={isOpen} className="menu-panel" t>
+      <Spacer>
+        <StyledDismissButton onClick={closeFunction} />
+      </Spacer>
+      <StyledChildrenContainer inline={context.inline}>
+        {children}
+      </StyledChildrenContainer>
+    </StyledPanel>
   );
 };
 
 MenuPanel.propTypes = {
   children: PropTypes.any.isRequired,
   isOpen: PropTypes.bool,
-  closeFunction: PropTypes.func.isRequired
-};
-
-MenuPanel.defaultProps = {
-  isOpen: false
+  closeFunction: PropTypes.func.isRequired,
```

Comment:
`onClose` might be a more appropriate name here. Has this already been published with this prop name?